embedded-kafka/src/test/scala/net/manub/embeddedkafka/EmbeddedKafkaMethodsSpec.scala

```diff
@@ -44,6 +44,9 @@ class EmbeddedKafkaMethodsSpec extends EmbeddedKafkaSpecSupport with EmbeddedKaf
 
       consumer.close()
 
+      // Commit message in embedded-kafka-spec group so we don't consume it
+      // again in the consumeFirstStringMessageFrom tests below.
+      consumeFirstStringMessageFrom(topic)
```

Comment:
Committing here shouldn't happen with using the `consumeFirstStringMessageFrom()` method, but with a direct commit instead.